v3

```diff
@@ -0,0 +1 @@
+.
```

Comment:
Could you remove this file?

Reply:
hi @terrytangyuan, the v3 file came from the output of the `make pre-commit -B` command the pull request instructions said to run first. I can remove though 👍